spynnaker8/models/model_data_holders/if_curr_alpha_data_holder.py

```diff
@@ -0,0 +1,49 @@
+from spynnaker.pyNN.models.neuron import AbstractPopulationVertex
+from spynnaker8.utilities import DataHolder
+from spynnaker.pyNN.models.neuron.builds import IFCurrAlpha
+
+
+class IFCurrAlphaDataHolder(DataHolder):
+    def __init__(
+            self,
+
+            spikes_per_second=AbstractPopulationVertex.
+            none_pynn_default_parameters['spikes_per_second'],
+
+            ring_buffer_sigma=AbstractPopulationVertex.
+            none_pynn_default_parameters['ring_buffer_sigma'],
+
+            incoming_spike_buffer_size=AbstractPopulationVertex.
+            none_pynn_default_parameters['incoming_spike_buffer_size'],
+
+            constraints=AbstractPopulationVertex.none_pynn_default_parameters[
+                'constraints'],
+
+            label=AbstractPopulationVertex.none_pynn_default_parameters[
+                'label'],
+
+            v_init=IFCurrAlpha.non_pynn_default_parameters['v_init'],
```

Comment:
All looks good to me.  I note that the correct spelling of "non_pynn" has been used here; personally I wouldn't bother changing the others though, as when we drop PyNN0.7 support we'll also be dropping any need to use data holders.